taskcluster/ci/csharp-build/kind.yml

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+---
+loader: glean_taskgraph.loader.build_config:loader
+
+transforms:
+  - glean_taskgraph.transforms.module_build:transforms
+  - taskgraph.transforms.job:transforms
+  - taskgraph.transforms.task:transforms
+
+job-defaults:
+  attributes:
+    run-on-pr-type: full-ci
+  run-on-tasks-for: [github-pull-request, github-push]
+  description: "{module_name} - Build and test"
```

Comment:
Is there any documentation for the TC syntax? What's `module_name` here?

Reply:
`{...}` is just python string interpolation.
`module_name` is only defined for `module_build` transforms here: https://github.com/mozilla/glean/blob/main/taskcluster/glean_taskgraph/transforms/module_build.py#L24

`module_builds` are the different modules for Gradle though, so it doesn't apply here.

* Remove the `transform` line above
* Set a static name here.